test/test-pipe-bind-error.c

```diff
@@ -137,3 +137,23 @@ TEST_IMPL(pipe_listen_without_bind) {
   MAKE_VALGRIND_HAPPY();
   return 0;
 }
+
+TEST_IMPL(pipe_bind_or_listen_error_after_close) {
+  uv_pipe_t server;
+  int r;
+
+  r = uv_pipe_init(uv_default_loop(), &server, 0);
```

Comment:
There are a number of cases where you're only using the return value in an assertion on the following line. Those can be consolidated a bit. Take a look at `ASSERT_EQ()` and similar assertions in the codebase.

Reply:
Done.